otx/algorithms/detection/adapters/mmdet/models/heads/custom_fcn_mask_head.py

```diff
@@ -0,0 +1,107 @@
+"""CustomFCNMaskHead for OTX template."""
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import torch
+from mmdet.models.builder import HEADS
+from mmdet.models.roi_heads.mask_heads.fcn_mask_head import FCNMaskHead
+
+from otx.algorithms.common.adapters.mmdeploy.utils import is_mmdeploy_enabled
+
+
+@HEADS.register_module()
+class CustomFCNMaskHead(FCNMaskHead):
+    """Custom FCN Mask Head for fast mask evaluation."""
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def get_seg_masks(self, mask_pred, det_bboxes, det_labels, rcnn_test_cfg, ori_shape, scale_factor, rescale):
+        """Get segmentation masks from mask_pred and bboxes.
```

Comment:
I think this function is changed to make it faster, right?
How about to explain which one is changed briefly to make maintenance easier later?

Reply:
I added this to the method: 

> The original `FCNMaskHead.get_seg_masks` grid sampled 28 x 28 masks to the original image resolution.
> As a result, the resized masks occupy a large amount of memory and slow down the inference.
> This method directly returns 28 x 28 masks and resizes to bounding boxes size in the post-processing steps.
> Doing so can save memory and speed up the inference.